### PR TITLE
Install common system library debugsymbols for unit tests.

### PR DESCRIPTION
**Pull request by kaidokert**
This helps symbolizing crashes that are occuring in the graphics and media libraries.

b/312772759

### CODE REVIEW

File: docker/linux/unittest/Dockerfile [hunk 1]
```diff
@@ -33,6 +37,15 @@ RUN apt update -qqy \
         unzip \
         xauth \
         xvfb \
+        libasound2-dbgsym \
+        # libavcodec58-dbgsym \
+        # libavformat58-dbgsym \
```
Comment (oxve) [inline docker/linux/unittest/Dockerfile:42]: Why are these two commented out?

File: docker/linux/unittest/Dockerfile [hunk 2]
```diff
@@ -16,6 +16,10 @@ FROM cobalt-base
 
 ARG HOME=/root
 
+RUN echo deb http://deb.debian.org/debian-debug \
+    $(. /etc/os-release && echo "$VERSION_CODENAME")-debug main \
```
Comment (oxve) [inline docker/linux/unittest/Dockerfile:20]: Env var references should be wrapped in curly braces:`${VERSION_CODENAME}`

**Review by oxve — APPROVED** [review]

File: docker/linux/unittest/Dockerfile (hunk 1 above)
Comment (kaidokert) [reply docker/linux/unittest/Dockerfile]: Removed. I couldn't figure out how to get the correct sources.list upstream for this ( yet ). There's some version mismatch.

But those are also not critical for what i need right now.

File: docker/linux/unittest/Dockerfile (hunk 2 above)
Comment (kaidokert) [reply docker/linux/unittest/Dockerfile]: Amended

**Review by isarkis — APPROVED** [review]

**Pull request merged**